src/main/webapp/features.jsp

```diff
@@ -2,187 +2,216 @@
+                <div class="col-xs-12 col-sm-9">
+                    <div class="features-page-text">
+                        Students can submit responses and view published responses using the unique links TEAMMATES emails them, without having to login or signup.
+                        <br><br>
+                        If they login to TEAMMATES using their Google accounts (optional), they can access all TEAMMATES courses in one page and access even more features such as setting up profiles.
+                    </div>
                 </div>
-                <img class="img-responsive" src="images/raised-edge.png">
+                <img class="features-page-img" src="images/raised-edge.png">
             </div>
         </section>
 
         <section class="row">
             <div class="col-xs-12 col-sm-8 col-sm-offset-2">
                 <h2 class="color_orange col-xs-12">Downloadable data</h2>
-                <img class="img-responsive col-xs-12" src="images/feature-downloadbledata.png" alt="Downloadable data">
-                <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-                    You can download the collected responses (and statistics) as spreadsheets.
-                    <br><br>
-                    The collected data belongs to you and you may delete your data from TEAMMATES any time.
-                </div>
-                <img class="img-responsive" src="images/raised-edge.png">
+                <img class="features-page-img" src="images/feature-downloadbledata.png" alt="Downloadable data">
+                <div class="col-xs-12 col-sm-9">
+                    <div class="features-page-text">
+                        You can download the collected responses (and statistics) as spreadsheets.
+                        <br><br>
+                        The collected data belongs to you and you may delete your data from TEAMMATES any time.
+                   </div>
+               </div>
+                <img class="features-page-img" src="images/raised-edge.png">
             </div>
         </section>
 
         <section class="row">
             <div class="col-xs-12 col-sm-8 col-sm-offset-2">
                 <h2 class="color_orange col-xs-12">Shareable comments</h2>
-                <img class="img-responsive col-xs-12" src="images/feature-sharablecomments.png" alt="Sharable">
-                <div class="col-xs-12 col-sm-10 col-sm-offset-1">
-                    You can add comments about responses collected. You can even share these comments with students/instructors.
-                    <br><br>
-                    Fine-grained visibility control is available for comments too.
-                    For example, you can give a comment to a student that the receiving student can see while other students in the course can only see the comment text but cannot see the recipient name.
+                <img class="features-page-img" src="images/feature-sharablecomments.png" alt="Sharable">
```

Comment:
I was able to get it all done, except for the the text alignment on smaller screens, as my lowest screen resolution (800x600) does not show the misalignment. I'm not sure how to setup a testing area for it either. Please assist.